src/App.tsx

```diff
@@ -1,61 +1,165 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+
+enum ChangeType {
+  TITLE,
+  USER,
+  COMPLETED,
+}
+
+export const App: React.FC<{}> = () => {
+  const [title, setTitle] = useState('');
+  const [user, setUser] = useState(0);
+  const [completed, setCompleted] = useState(false);
+  const [todos, setTodos] = useState(
+    todosFromServer.sort((a, b) => a.id - b.id),
```

Comment:
let's not caclulate the state inside the useState, better to make it a separate variable